src/traci/PosixLauncher.ned

```diff
@@ -9,4 +9,7 @@ simple PosixLauncher like Launcher
         string sumocfg;
         int port = default(0);
         int seed = default(23423);
+
+        // additional SUMO command line options
+        string options = default("");
```

Comment:
Hm, `port` and `seed`are "options" as well. Would "extra" or something similar be a better name?

Reply:
I feel "extra" is not expressive enough.
I've changed it to "extraOptions", since these options are not mandatory.